### ISSUE TITLE
Logging control?

### ISSUE THREAD

**Issue by DoumanAsh** [aatxe/irc#105]
Currently it seems to dump all messages to stdout, but since irc doesn't use any logging library there is no control of it?

If there is no control on logging it would be useful to provide some way for it?

**Comment by aatxe** [comment]
I agree that we should use logging (via `log` crate, most likely), and eliminate any printlns. Did you actually encounter any output though? A cursory search seems to turn up only one println that is not inside of a test case, and that would appear only once in a program. In general, I tried to avoid making the library noisy (with the understanding that there is no logging framework). There is probably places where it would be nice to log info (especially when #88 is done).

**Comment by DoumanAsh** [comment]
@aatxe Yeah sorry, at first look I thought there are several printlns, but then I examined in details and too found only one rare println on adding certificate path.

To me of course silent irc is better than anything else :)
